drizzlepac/haputils/product.py

```diff
@@ -127,6 +129,220 @@ def generate_metawcs(self):
 
         return meta_wcs
 
+    def align_to_gaia(self, catalog_list=[], headerlet_filenames=None, output=True,
```

Comment:
Input parameter 'headerlet_filenames' is never used, and gets overwritten on Line 144 even if it was provided as input.  So, this parameter can be removed as an input parameter.

Reply:
Removed the unnecessary parameter in c163f8c.